src/engraving/rendering/score/harmonylayout.cpp

```diff
@@ -332,6 +332,13 @@ void HarmonyLayout::render(Harmony* item, Harmony::LayoutData* ldata, const Layo
     ldata->fontList.mut_value().clear();
     for (const ChordFont& cf : chordList->fonts) {
         Font ff(item->font());
+        ff.setPointSizeF(item->size());
+        ff.setFamily(item->family(), Font::Type::Harmony);
+        ff.setBold(item->bold());
+        ff.setItalic(item->italic());
+        ff.setUnderline(item->underline());
+        ff.setStrike(item->strike());
```

Comment:
Aren't these already correct in `item->font()`, because that calls those virtual methods?